### ISSUE TITLE
Colab example no longer works out of the box

### ISSUE THREAD

**Issue by acekrystal** [ultralytics/yolov5#4749]
I just tried to use the colab example : https://colab.research.google.com/github/pytorch/pytorch.github.io/blob/master/assets/hub/ultralytics_yolov5.ipynb
on the website : https://pytorch.org/hub/ultralytics_yolov5/

But the Colab gives me multiple issues : 
issue 1. Running the first line : 
```
%%bash
pip install -qr https://raw.githubusercontent.com/ultralytics/yolov5/master/requirements.txt  # install dependencies

```
Gives me : 

```
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
albumentations 0.1.12 requires imgaug<0.2.7,>=0.2.5, but you have imgaug 0.2.9 which is incompatible.

```

issue 2. The second line : `import torch`
Gives me an error at some point : 

```
Model Summary: 224 layers, 7266973 parameters, 0 gradients
Adding AutoShape... 
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-2-0a0ec081fe3a> in <module>()
      8 
      9 # Inference
---> 10 results = model(imgs)
     11 
     12 # Results

7 frames
/usr/local/lib/python3.7/dist-packages/PIL/TiffImagePlugin.py in ImageFileDirectory_v2()
    685                 (TiffTags.FLOAT, "f", "float"),
    686                 (TiffTags.DOUBLE, "d", "double"),
--> 687                 (TiffTags.IFD, "L", "long"),
    688             ],
    689         )

AttributeError: module 'PIL.TiffTags' has no attribute 'IFD'

```

Underneath it stated that "Issues should be raised directly in https://github.com/ultralytics/yolov5." So here I did as I did not see other issues made yet about so far my search showed. :)

**Comment by glenn-jocher** [comment]
@acekrystal if you are using colab you may need to restart your runtime after installing packages. You can do this using Runtime > Restart Runtime or programmatically using `exit()`.

**Comment by glenn-jocher** [comment]
@acekrystal good news 😃! Your original issue may now be fixed ✅ in PR #4842. This relaxes Pillow dependency requirements that were causing the issue. Colab notebook should now work directly the first time with no issues!

To receive this update:
- **[Git](https://github.com/ultralytics/yolov5)** – `git pull` from within your `yolov5/` directory or `git clone https://github.com/ultralytics/yolov5` again
- **[PyTorch Hub](https://pytorch.org/hub/ultralytics_yolov5/)** – Force-reload with `model = torch.hub.load('ultralytics/yolov5', 'yolov5s', force_reload=True)`
- **[Notebooks](https://github.com/ultralytics/yolov5/blob/master/tutorial.ipynb)** – View updated notebooks  <a href="https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/ultralytics/yolov5"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **[Docker](https://hub.docker.com/r/ultralytics/yolov5)** – `sudo docker pull ultralytics/yolov5:latest` to update your image <a href="https://hub.docker.com/r/ultralytics/yolov5"><img src="https://img.shields.io/docker/pulls/ultralytics/yolov5?logo=docker" alt="Docker Pulls"></a>

Thank you for spotting this issue and informing us of the problem. Please let us know if this update resolves the issue for you, and feel free to inform us of any other **issues** you discover or **feature requests** that come to mind. Happy trainings with YOLOv5 🚀!
